gulpfile.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
```

Comment:
Since it is an Electron app, it is built on Node too, so this editor is perfectly adapted to Node development.